avalanche-ops/src/aws/cfn-templates/vpc.yaml

```diff
@@ -273,6 +285,15 @@ Resources:
       FromPort: !Ref HttpPort
       ToPort: !Ref HttpPort
       CidrIp: !Ref HttpPortIngressIpv4Range
+  
+  UserDefinedIngress:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      GroupId: !Ref SecurityGroup
```

Comment:
We need to define a "new" security group. The current changes opens the port to the "node", which we don't want. And use the new security group (sg) here